README.rst

```diff
@@ -105,9 +105,13 @@ settings: ``lms/envs/common.py``
    # helps to configure probability of displaying the verification forms. Values in range 0 to 1 are allowed, where 0
    # means never and 1 means always display. Default value is 0.5 i.e. 50% chance of displaying the form.
    SHOW_SKILL_VERIFICATION_PROBABILITY = 0.5
-   # Optionally update topic name for verification event emitted when a user verifies tags for an xblock.
-   EVENT_BUS_XBLOCK_VERIFICATION_TOPIC = "learning-custom-xblock-skill-verfied"
-
+   # Include the XBLOCK_SKILL_VERIFIED signal in `EVENT_BUS_PRODUCER_CONFIG` to push the event to event bus
+   # Note: Include `openedx_events` in `INSTALLED_APPS` setting in the host application
+   EVENT_BUS_PRODUCER_CONFIG = {
+       'org.openedx.learning.xblock.skill.verified.v1': [
+           {'topic': 'learning-custom-xblock-skill-verfied', 'event_key_field': 'xblock_info.usage_key', 'enabled': True},
```

Comment:
I believe the topic name is not correct. This is the name I see on Confluent for different environments:
PROD: `prod-learning-xblock-skill-verified`
STAGE: `stage-learning-xblock-skill-verified`

Reply:
@sameenfatima78 The env prefix should not be a part of the topic name, it is set via `EVENT_BUS_TOPIC_PREFIX` setting as documented [here](https://github.com/openedx/openedx-events/blob/main/docs/how-tos/adding-events-to-event-bus.rst#producing-to-event-bus).

Reply:
Sorry for not being clear but I wasn't talking about the prefix part. I see `learning-custom-xblock-skill-verfied` but topic name that we have set up on Confluent is `learning-xblock-skill-verified`

Reply:
@sameenfatima78 Ohh, got it. I'll update the example here.